vmdk_plugin/utils/fs/fs.go

```diff
@@ -37,6 +37,9 @@ const pciAddrLen = 10                     // Length of PCI dev addr
 // FstypeDefault contains the default FS when not specified by the user
 const FstypeDefault = "ext4"
 
+// BinSearchPaths contains search paths for host binaries
+var BinSearchPaths = [4]string{"/bin", "/sbin", "/usr/bin", "/usr/sbin"}
```

Comment:
This may not be needed here, why not have a "default" search path in the default config used by the plugin and that can be used as the search path for all external execs that the plugin runs. The plugin behavior can be changed any time.